src/static/riot/datasets/management.tag

```diff
@@ -383,6 +382,22 @@
                     self.hide_progress_bar()
                 })
         }
+
+        self.toggle_is_public = () => {
```

Comment:
I think it's maybe worth mentioning that we _could _ abstract out most of this `toggle_public` function into some kind of riot mixin, but I don't think the effort is worth it here.

More info if we ever see a need for this in the future:
https://v3.riotjs.now.sh/guide/#mixins